db/schema.rb

```diff
@@ -114,7 +114,9 @@
     t.string "iban"
     t.string "ip_hash"
     t.boolean "account_active", default: false, null: false
+    t.bigint "parent_id"
     t.index ["mobile_phone"], name: "index_people_on_mobile_phone", unique: true
+    t.index ["parent_id"], name: "index_people_on_parent_id"
```

Comment:
TODO: the version number in the schema didn't update because I made the migration after the current migration. So rename the migration so that it is after the current one, so the migration actually runs on currently running systems.

Reply:
Fixed.